Dockerfile

```
@@ -1,4 +1,4 @@
-FROM python:3.8
+FROM python:3.8-slim
 
 WORKDIR /usr/src/app
 
```

```suggestion
RUN apt-get update && apt-get install -y build-essential libcurl4-openssl-dev libssl-dev
```

Needs these packages, otherwise works great